boromir674/cookiecutter-python-package
# Cookiecutter Python Package v1.5.1 Release

1.5.1 (2022-06-20)
==================

Changes
-------

The Update includes improved code Architecture and better Test code Coverage!
Features shorter Unites of Code, more DRY and more Tests.

test
""""
- refactor tests, add bandit tox env
- verify commit message, author and email are the expected ones
- git init, increase test code coverage

refactor
""""""""
- reduce code
- centralize `subprocess`run invocations

ci
""
- enable network-bound tests to trigger integration testing of the `check_pypi` feature
